actors/abi/sector.go

```diff
@@ -22,6 +22,10 @@ func (s SectorNumber) String() string {
 // Raising this would require modifying our AMT implementation.
 const MaxSectorNumber = math.MaxInt64
 
+// MaxPartitionsPerDeadline is the maximum number of partitions that will be assigned to a deadline.
+// The current value allows upto 1 Eib storage for a miner.
+const MaxPartitionsPerDeadline = 300
```

Comment:
This is in the wrong place, it should be in `miner/policy.go`.

Also this value is way too small. 1EiB is a _minimum_ to support.

Note: since this number is derived from other numbers, that might change, we should do the arithmetic here either in comment, or, better, the value itself (#874). But if we set it to a value way higher, that's less important.

I propose:
- Document that `48 * 32GiB * 2349 * 300 = 1.00808144 EiB` ([link](https://www.google.com/search?q=48+*+32GiB+*+2349+*+300+in+EiB)) 
- Set this to 3000 (i.e. 10EiB)

If you use units of GiB, you can probably do the arithmetic with int64s to derive this in place.

Reply:
If we set this to 3000, we'll need to process up to 3000 partitions at the end of a deadline. I wonder if we should bump the max addressed partitions limit to match.

Reply:
> Note: since this number is derived from other numbers, that might change, we should do the arithmetic here either in comment, or, better, the value itself (#874).

Really, there are two constraints here:

1. Maximum supported miner size. This is set as a product requirement.
2. Maximum number of partitions per deadline. This is bounded above by how much work we can do per epoch, and below by how many partitions we need to support 1.

Reply:
> bump the max addressed partitions limit to match

Yes, that seems fair.

Reply:
@Stebalien What is this "ax addressed partitions" that you speak of ?

Reply:
This is done.